### ISSUE TITLE
unable to insert new spools mask is not showing up

### ISSUE THREAD

**Issue by sniper22nd** [OllisGit/OctoPrint-SpoolManager#181]
![image](https://user-images.githubusercontent.com/85032133/120078468-05433600-c0b0-11eb-8624-bd1da656be3d.png)

**Comment by erwinf** [comment]
I also encountered the same issue

**Comment by gn0m3r4l** [comment]
Same for me.
Plugin Version 1.4.3
Octoprint 1.6.1

Unfortunately, I cannot tell when this occurs first time.

**Comment by moorsey** [comment]
Same as #172 I think

**Comment by sniper22nd** [comment]
Yes I do, however it seems to be working OK at the moment Sent from Android deviceAm 19.10.2021 00:29 schrieb OllisGit ***@***.***>:
Hi @sniper22nd, @gn0m3r4l, @erwinf
do you use the plugin UI Customizer?
